book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] I suggest that you make this `attr_accessor: rentals`  to be able to access it and modify it